subsys/bluetooth/controller/ll_sw/nordic/lll/lll_df.c

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdint.h>
+#include "hal/radio_df.h"
+#include "lll_df.h"
+
+/* @brief Function performs common steps for initialization and reset
+ * of Direction Finding LLL module.
+ *
+ * @return	Zero in case of success, other value in case of failure.
+ */
+static int init_reset(void);
+
+/* @brief Function performs Direction Finding initialization
+ *
+ * @return	Zero in case of success, other value in case of failure.
+ */
+int lll_df_init(void)
+{
+	int err;
+
+	err = radio_df_ant_configure();
+	if (err) {
+		return err;
+	}
+
+	err = init_reset();
+	if (err) {
+		return err;
+	}
+
+	return 0;
+}
+
+
+/* @brief Function performs Direction Finding reset
+ *
+ * @return	Zero in case of success, other value in case of failure.
+ */
+int lll_df_reset(void)
+{
+	int err;
+
+	err = init_reset();
+	if (err) {
+		return err;
+	}
+
+	return 0;
+}
+
+static int init_reset(void)
+{
+	return 0;
+}
+
```

Comment:
Minor nitpick... move this to after non file static functions (to the end of this file).

Reply:
function moved to bottom of the file